app/controllers/letter_opener_web/letters_controller.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 module LetterOpenerWeb
-  class LettersController < ApplicationController
+  class LettersController < LetterOpenerWeb::ApplicationController
```

Comment:
I don't see any harm with this approach, but I think I'd prefer adding the `require_dependency` back over explicitly setting the namespace here.

Reply:
@koheisg can you update your PR to add back the `require_dependency` or would you prefer I open a new one?